hardware/OpenWebNetTCP.cpp

```diff
@@ -1097,6 +1097,8 @@ bool COpenWebNetTCP:: WriteToHardware(const char *pdata, const unsigned char len
 bool COpenWebNetTCP::sendCommand(bt_openwebnet& command, vector<bt_openwebnet>& response, int waitForResponse, bool silent)
 {
     csocket *commandSocket;
+	bool ret;
```

Comment:
I think is not necessary, the variable is always set before being used.
In one case from ParseData() function if you expect the response to the command, in the other case it is always set to true.

However if you want I can add to the default false value ....